### ISSUE TITLE
[CodeBlock]: Hyperlink missing link text or image with alt text

### ISSUE THREAD

**Issue by jendowns** [carbon-design-system/gatsby-theme-carbon#572]
The `CodeBlock` component uses an anchor in its inner `PathRow` and `Sidebar` subcomponents that wraps an svg. This anchor has a `title` attribute, but the inner svg needs an `alt` as well:

![Screen Shot 2019-11-26 at 9 54 48 AM](https://user-images.githubusercontent.com/9057921/69649857-4d087d80-1033-11ea-9ff0-786c3ee1a2c1.png)

The DAP tool link includes a very relevant example as well:

```html
Example of proper code:
This example shows a hyperlink containing an image with alternative text:

<a title="Print this page" href="print.htm">
  <img src="img.png" alt="Print this page">
</a>
```

##  What WCAG 2.1 checkpoint does the issue violate?

* [Checkpoint 1.1.1 Non-Text Content](https://www.ibm.com/able/guidelines/ci162/non_text_content.html)
* [Checkpoint 2.4.4 Link Purpose (in Context)](https://www.ibm.com/able/guidelines/ci162/link_purpose.html)

**Comment by jendowns** [comment]
I can propose a fix for this 👍
